backend/server/src/db/cmd.rs

```diff
@@ -10,6 +17,10 @@ use super::{Db, create_pool, query};
 
 
 pub(crate) async fn run(cmd: &DbCommand, config: &config::Db) -> Result<()> {
+    if let DbCommand::Console = cmd {
+        return console(config).map(|_| ())
```

Comment:
```suggestion
        return console(config).map(|_| ());
```

Sorry :/ But I think a semicolon is usually expected behind return, even if not strictly necessary.

Reply:
No problem, done. :D